### DISCUSSION TITLE
Audiogridder causing latency

### DISCUSSION THREAD

**Discussion by sdd1061** [apohl79/audiogridder#596] [Help]
When I insert audiogridder and load a plugin it creates latency on the audio track!

Macpro 6core 3.5 128ram 2013 trashcan
server is a macbookpro 2020 16" 64ram 
cubase 11 latest build

any suggestions are helpful
thank you vm
Stephen

**Comment by lesbrunn** [comment]
Are you using VI's? A setting of 0 blocks has less latency, but causes crashes for me. 1 block seems to work better, but that also causes breaks in the audio. Generally though, AG may be better suited for mixing, where latency wouldn't be so much of an issue. 

In comparison, I believe VE Pro has very efficient internal latency compensation, which makes it smoother with live and VI use. Most of the latency is taken care of before it hits the DAW.
